packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -54,6 +54,22 @@ export interface ClusterProps {
    */
   readonly capacityProviders?: string[];
 
+  /**
+   * The cluster default capacity provider strategy. This takes the form of a list of CapacityProviderStrategy objects.
+   *
+   * For example
+   * [
+   *  {
+   *    capacityProvider: 'FARGATE',
+   *    base: 10,
+   *    weight: 50
+   *  }
+   * ]
+   *
+   * @default null
+   */
+  readonly defaultCapacityProviderStrategy?: CapacityProviderStrategy[];
```

Comment:
Can you help me understand why this takes a list instead of just a single value?

Reply:
As we know that Capacityprovider contains Name,Base and weight, Instead of providing single value if you provide list. It's possible to create more complicated capacity providers strategies than just by specifying fargate as a string for the capacityprovider.